src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -395,7 +395,7 @@ def _generate_run_name(self) -> str:
     def _get_default_display_name(self) -> str:
         display_name = self.display_name
         if not display_name:
-            display_name = self._get_flow_dir().name
+            display_name = self.name
```

Comment:
can simple use `display_name = self.display_name or self.name`?

Reply:
Let me fix in next PR.